static/app/components/group/sidebar.tsx

```diff
@@ -68,12 +65,6 @@ class BaseGroupSidebar extends Component<Props, State> {
     this.fetchTagData();
   }
 
-  UNSAFE_componentWillReceiveProps(nextProps: Props) {
-    if (!isEqual(nextProps.environments, this.props.environments)) {
-      this.setState({environments: nextProps.environments}, this.fetchTagData);
```

Comment:
oh you know what, maybe the `fetchTagData` still needs to happen 😬

Reply:
`componentDidUpdate` for that fetch